src/components/chart/_macro.njk

```diff
@@ -1,7 +1,7 @@
 {% from "components/list/_macro.njk" import onsList %}
 
 {% macro onsChart(params) %}
-    {% set supportedChartTypes = ['line', 'bar', 'column'] %}
+    {% set supportedChartTypes = ['line', 'bar', 'column', 'scatter', 'area'] %}
```

Comment:
```suggestion
    {% set supportedChartTypes = ['line', 'bar', 'column', 'scatter'] %}
```

I know its only a small change but I do think we should remove this to keep changes separate from each other. It helps to see what was changed if we need to revisit these PRs at some point

Reply:
I actually think we should have all charts we planned to support in from the start... but happy to remove it/

Reply:
Updated